### PR TITLE
Create a DBus proxy of the DNF module

### PR DESCRIPTION
**Pull request by poncovka**
An instance of class DNFPayload can access its DBus counterpart via the property
proxy. The property will be eventually moved to the class Payload and created
for each type of the payload.

Refactorize the DBus API for working with payload modules. The current design
is based on the API for the partitioning modules.

### CODE REVIEW

File: pyanaconda/ui/lib/payload.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#
```
Comment (jkonecny12) [inline pyanaconda/ui/lib/payload.py:1]: I think this code should be somewhere in the `/pyanaconda/payload` not UI code.

File: pyanaconda/ui/lib/payload.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    object_path = payloads_proxy.CreatePayload(payload_type)
+
+    if activate:
+        payloads_proxy.ActivatePayload(object_path)
+
+    return PAYLOADS.get_proxy(object_path)
+
+
+def get_payload(payload_type=PAYLOAD_TYPE_DNF):
+    """Get a payload of the specified type.
```
[the remark this answers is not in the document]
Comment (poncovka) [reply pyanaconda/ui/lib/payload.py:42]: Fixed.

File: pyanaconda/ui/lib/payload.py (hunk 1 above)
Comment (poncovka) [reply pyanaconda/ui/lib/payload.py:1]: `pyanaconda.payload` will be removed. We usually put these functions to `pyanaconda.core` when they are shared with modules or to `pyanaconda.ui.lib` when they are used only by the UI. Look at `pyanaconda.ui.lib.storage` and `pyanaconda.ui.lib.users`.

Comment (poncovka) [reply pyanaconda/ui/lib/payload.py:1]: It is true, that we might want to merge `pyanaconda.ui.lib` with `pyanaconda.core`.

Comment (jkonecny12) [reply pyanaconda/ui/lib/payload.py:1]: OK, thanks for explanation.

**Pull request merged**